draft-ietf-scitt-architecture.md

```diff
@@ -760,9 +760,9 @@ An Issuer must first decide on a suitable format to serialize the Statement payl
 - in-toto
 - SLSA
 
-Once the Statement is serialized with the correct media-type/content-format, an Issuer should fill in the attributes for the Registration Policy information header.
+Once the Statement is serialized with the correct media-type/content-format, an Issuer MUST fill in the attributes for the Registration Policy information header.
```

Comment:
we should consider "Statement Issuer" vs "Receipt Issuer", for better clarity.

Reply:
Now captured in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/97